src/framework/theme/components/radio/radio-group.component.ts

```diff
@@ -277,4 +273,12 @@ export class NbRadioGroupComponent implements AfterContentInit, OnDestroy, Contr
       });
     }
   }
+
+  protected getUpdateCheckedValue() {
```

Comment:
Let's come up with a more descriptive method name. What do you think about `updateValueFromCheckedOption`?